pdc/apps/unreleasedvariant/views.py

```diff
@@ -21,6 +21,7 @@ class UnreleasedVariantViewSet(viewsets.PDCModelViewSet):
     filter_class = UnreleasedVariantFilter
     serializer_class = UnreleasedVariantSerializer
     lookup_field = 'uid'
+    lookup_value_regex = r'[^/]+'
```

Comment:
Can this be moved to some base class? And ideally use constant with descriptive name or add comment so this is not accidentally removed again.